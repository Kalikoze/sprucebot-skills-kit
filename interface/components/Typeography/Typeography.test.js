import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

describe('p tag', () => {
	const { Paragraph } = require('./Typeography')

	test('it renders', () => {
		const tree = renderer.create(<Paragraph />).toJSON()
		expect(tree).toMatchSnapshot()
		expect(tree).toHaveStyleRule('font-size', '1em')
	})
})

describe('h1 tag', () => {
	const { H1 } = require('./Typeography')
	test('it renders', () => {
		const tree = renderer.create(<H1 />).toJSON()
		expect(tree).toMatchSnapshot()
		expect(tree).toHaveStyleRule('font-size', '1.5em')
	})
})
