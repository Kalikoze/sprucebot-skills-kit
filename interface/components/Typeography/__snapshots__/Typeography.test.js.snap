// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`h1 tag it renders 1`] = `
.c0 {
  font-size: 1.5em;
}

<h1
  className="c0"
/>
`;

exports[`p tag it renders 1`] = `
.c0 {
  font-size: 1em;
}

<p
  className="c0"
/>
`;
